ReadMe.md

```diff
@@ -3,7 +3,8 @@
 
 [![AppVeyor build status windows](https://img.shields.io/appveyor/job/build/yaxlib/yaxlib/windows/master?label=windows%20build)](https://ci.appveyor.com/project/yaxlib/yaxlib/branch/master)
 [![AppVeyor build status linux](https://img.shields.io/appveyor/job/build/yaxlib/yaxlib/linux/master?label=linux%20build)](https://ci.appveyor.com/project/yaxlib/yaxlib/branch/master)
-[![codecov](https://codecov.io/gh/YAXLib/YAXLib/branch/master/graph/badge.svg?token=9VHIEiyUt8)](https://codecov.io/gh/YAXLib/YAXLib) 
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=yaxlib_YAXLib&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=yaxlib_YAXLib)
```

Comment:
FYI I like this one, but we could also choice other badges. See https://sonarcloud.io/project/information?id=yaxlib_YAXLib

Reply:
Indeed, we should integrate SonarCloud, which is not limited to a single badge.
I also like this one:
[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=yaxlib_YAXLib&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=yaxlib_YAXLib)
Make your choice of the badge used for integration ;-)